doc/movingparts.rst

```diff
@@ -110,11 +105,11 @@ You can alter the stream content with filters provided by html5lib:
   the document
 
 * :class:`lint.Filter <html5lib.filters.lint.Filter>` raises
-  ``LintError`` exceptions on invalid tag and attribute names, invalid
+  :exc:`AssertionError` exceptions on invalid tag and attribute names, invalid
```

Comment:
Is it really an AssertionError? If so, we should write up an issue to change that.

Reply:
Yeah, the implementation is basically all `assert` statements: https://github.com/html5lib/html5lib-python/blob/7bbde541e08ca82c07af5ac7e5f64525f1793814/html5lib/filters/lint.py#L24-L79